Numsense/Chinese.fs

```diff
@@ -0,0 +1,194 @@
+﻿module internal Ploeh.Numsense.Chinese
+
+open Ploeh.Numsense.InternalDsl
+
+type CharacterSet =
+    {
+        negative: string;
+        zero: string;
+        one: string;
+        two: string;
+        alternativeTwo: string;
+        three: string;
+        four: string;
+        five: string;
+        six: string;
+        seven: string;
+        eight: string;
+        nine: string;
+        ten: string;
+        hundred: string;
+        thousand: string;
+        tenThousands: string;
+        hundredMillions: string
+    }
+
+let internal TraditionalCharacterSet =
+    {
+        negative = "負";
+        zero = "零";
+        one = "一";
+        two = "二";
+        alternativeTwo = "兩";
+        three = "三";
+        four = "四";
+        five = "五";
+        six = "六";
+        seven = "七";
+        eight = "八";
+        nine = "九";
+        ten = "十";
+        hundred = "百";
+        thousand = "千";
+        tenThousands = "萬";
+        hundredMillions = "億"
+    }
+
+let internal TraditionalFinancialCharacterSet =
```

Comment:
Since this is a value, it should be camelCased, i.e. `traditionalFinancialCharacterSet`.

Same reason as outlined above.